SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorCloudLauncher.cpp

```diff
@@ -6,14 +6,16 @@
 #include "SpatialGDKEditorSettings.h"
 #include "SpatialGDKServicesModule.h"
 
+DEFINE_LOG_CATEGORY(LogSpatialGDKEditorCloudLauncher);
+
 bool SpatialGDKCloudLaunch()
 {
 	const USpatialGDKEditorSettings* SpatialGDKSettings = GetDefault<USpatialGDKEditorSettings>();
 
-	const FString CmdExecutable = TEXT("cmd.exe");
+	const FString LauncherExe = FSpatialGDKServicesModule::GetSpatialGDKPluginDirectory(TEXT("SpatialGDK/Binaries/ThirdParty/Improbable/Programs/DeploymentLauncher/DeploymentLauncher.exe"));
```

Comment:
This should be a const in the class I believe.

Reply:
Moved to anonymous namespace above.